pydruid/client.py

```diff
@@ -383,8 +384,12 @@ class PyDruid(BaseDruidClient):
                 0      7  2013-10-04T00:00:00.000Z         user_1
                 1      6  2013-10-04T00:00:00.000Z         user_2
     """
-    def __init__(self, url, endpoint):
+    def __init__(self, url, endpoint, proxies = None):
         super(PyDruid, self).__init__(url, endpoint)
+        if proxies != None:
```

Comment:
would go `if proxies:` or `if proxies is not None:`, but `!=` with `None` is a no-no in python I think.

Reply:
I'm not good at pythonic-syntax. Thanks for comment.